pkg/kudoctl/cmd/init.go

```diff
@@ -41,6 +41,10 @@ To dump a manifest containing the KUDO deployment YAML, combine the '--dry-run'
   kubectl kudo init --client-only
   # init client for non-default home
   kubectl kudo init --client-only --home /opt/home2
+  # install kudo crds only
+  kubectl kudo init --crd-only
+  # delete crds
+  kubectl kudo init --crd-only --dry-run --output yaml | kubectl delete -f -
```

Comment:
nice, can we document also `kubectl kudo init --dry-run --output yaml | kubectl delete -f -` as a way to uninstall everything kudo from your cluster? I think that could be handy as well